.github/workflows/build-test.yaml

```diff
@@ -17,11 +17,16 @@ jobs:
         with:
           go-version: '1.17.6'
       - run: GOOS=linux GOARCH=amd64 go build -o artifacts/pipecd ./cmd/pipecd
```

Comment:
I think it better which we create make command and call that command from CI.
However we've already used `build` for bazel, it's up to you whether replace that command or not.

Reply:
Totally agree about that.
I will refactor it later once confirmed that it works for our case.